code/core/system/proxy-fh16/src/CANMessageDataStore.cpp

```diff
@@ -77,6 +78,10 @@ void CanMessageDataStore::add(odcore::data::Container &container) {
     m_enabled = true;
 
     if (container.getDataType() == opendlv::proxy::ActuationRequest::ID()) {
```

Comment:
Instead of reacting on a specific CAN message that could be changed, couldn't we simply forbid the access to candevice->write(...)?